.github/workflows/build_runtime_cmake.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+on:
+  workflow_call:
+    inputs:
+      build-dir:
+        required: true
+        type: string
+
+
+env:
+  IREE_DOCKER_WORKDIR: "/usr/src/github/iree"
+
+jobs:
+  build_runtime:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e  # v2
+        with:
+          submodules: true
+      - name: "Building runtime"
+        run: |
+          docker run --user="$(id -u):$(id -g)" \
+            --volume="$PWD:$IREE_DOCKER_WORKDIR" \
+            --workdir="$IREE_DOCKER_WORKDIR" \
+            --rm \
+            gcr.io/iree-oss/base@sha256:9d742e01507c292def852cbfebfae71412cff94df0ab2619f61f9a5a2a98f651 \
```

Comment:
Should we define the docker SHA version as an input variable? So it is easier to keep consistency between workflows.

Reply:
Maybe? We have scripting right now that replaces docker digests across the repo though, so I don't think it's high priority.